go/store/nbs/must_test.go

```diff
@@ -26,7 +29,7 @@ func mustUint64(val uint64, err error) uint64 {
 	return val
 }
 
-func mustAddr(h addr, err error) addr {
+func mustAddr(h hash.Hash, err error) hash.Hash {
```

Comment:
Nit: Unused? MaybeParse() returns a `bool` instead of the `err`, so the context in which I would expect to see this used must not have come up?

Reply:
Dead code. removing.